### PR TITLE
Initial set up for People-Service

### PR DESCRIPTION
**Pull request by bbchoe**
- set up repo
- can write user profiles to database via three methods:
1) user profile over HTTP
2) stream write to DB
3) bulk write to DB

### CODE REVIEW

**Comment by bbchoe** [comment]
- initial test suite built
- database is seeded with 10mm+ entries
- travis integration built

File: package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "model-instagram-people-service",
+  "version": "1.0.0",
+  "description": "Handle People-related requests for Model-Instagram clone",
+  "main": "./server/index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "start:server": "export NODE_ENV=production && nodemon --max_old_space_size=4000 ./server/index.js",
```
Comment (elena-cz) [inline package.json:10]: Suggest changing this to development and creating a separate production environment later

File: server/index.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+  console.log('environment variables ', process.env.NODE_ENV);
+  let lastUserId = Number(fs.readFileSync('./simulator/lastUserId.txt', 'utf8'));
+  if (process.env.NODE_ENV === 'test') {
+    lastUserId = 0;
+  }
+  console.log('First user ID is ', lastUserId);
+  const maxUserId = lastUserId + volOfUsersToAdd;
+
+  const insertUserToDb = (userId) => {
```
Comment (elena-cz) [inline server/index.js:133]: I would consider moving these database functions into a separate file

**Review by elena-cz — APPROVED** [review]
Code looks good, just left a couple comments

**Pull request merged**